### DISCUSSION TITLE
Welcome to ramp.library Discussions!

### DISCUSSION THREAD

**Discussion by smitdave** [dd-harp/ramp.library#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smitdave** [comment]
I wanted to flag up for @doreenmbabazi and @Jnsumba that I've started up discussions here on ramp.library.

**Reply by smitdave** [reply]
Are you two seeing this discussion thread?

**Comment by slwu89** [comment]
Hello, no longer an active developer, but happy to be on hand for discussions regarding extensions of work in the spatial dynamics paper. I am also highly interested in any applications of applied category theory to model construction, communication, etc.

**Reply by smitdave** [reply]
Hello, Sean. Nice to see you hanging out here.
